security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,47 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
+
+* Previously, the ACS central component was unaware if it received all the `ComplianceCheckResults` for a `DONE` status `ComplianceScan`. With this update, the ACS central component receives the number of expected `ComplianceCheckResults` for a `ComplianceScan` with a `DONE` status. (link:https://issues.redhat.com/browse/CMP-2615[*CMP-2615*])
+
+* Previously, the `ocp4-cis-scc-limit-container-allowed-capabilities` rule description contained ambiguous guidelines, leading to confusion among users. With this update, the rule description and actionable steps are clarified. (link:https://issues.redhat.com/browse/OCPBUGS-17828[*OCPBUGS-17828*])
+
+* Before this update, sysctl configurations caused certain auto remediations for RHCOS4 rules to fail scans in affected clusters. With this update, the correct sysctl settings are applied and RHCOS4 rules for FedRAMP High profiles pass scans correctly. (link:https://issues.redhat.com/browse/OCPBUGS-19690[*OCPBUGS-19690*])
+
+* Before this update, an issue with a `jq` filter caused errors with the `rhacs-operator-controller-manager` deployment during compliance checks. With this update, the `jq` filter expression is updated and the `rhacs-operator-controller-manager` deployment is exempt from compliance checks pertaining to container resource limits, eliminating false positive results. (link:https://issues.redhat.com/browse/OCPBUGS-19690[*OCPBUGS-19690*])
+
+* Before this update, `rhcos4-high` and `rhcos4-moderate` profiles checked values of an incorrectly titled configuration file. As a result, some scan checks could fail. With this update, the rhcos4 profiles now check the correct configuration file and scans pass correctly. (link:https://issues.redhat.com/browse/OCPBUGS-31674[*OCPBUGS-31674*])
+
+* Previously, the `accessokenInactivityTimeoutSeconds` variable used in the `oauthclient-inactivity-timeout` rule wass immutable, leading to a `FAIL` status when performing DISA STIG scans. With this update, proper enforcement of the `accessTokenInactivityTimeoutSeconds` variable operates correctly and a `PASS` status is now possible. (link:https://issues.redhat.com/browse/OCPBUGS-32551[*OCPBUGS-32551*])
+
+* Before this update, some annotations for rules were not updated, displaying the incorrect control standards. With this update, annotations for rules are updated correctly, ensuring the correct control standards are displayed. (link:https://issues.redhat.com/browse/OCPBUGS-34982[*OCPBUGS-34982*])
+
+* Previously, an outdated link to the documentation in the {product-title} web console impeded users from accessing the Compliance Operator documentation directly. With this update, the URL has been rectified and you can directly access the documentation from the web console. (link:https://issues.redhat.com/browse/OCPBUGS-37416[*OCPBUGS-37416*])
```

Comment:
The bug OCPBUGS-37416 was dropped from the errata due to not working as expected. Please drop the release note for it. Thanks. cc: @rhmdnd